src/components/Nav.jsx

```diff
@@ -1,10 +1,23 @@
 import { Link } from "react-router-dom";
+import { NavLink } from "react-router-dom";
 import H1 from "./atoms/H1";
+import H2 from "./atoms/H2";
 
 export default function Nav() {
   return (
-    <nav className="bg-yellow-500 p-4 flex justify-center md:justify-start">
+    <nav className="bg-yellow-500 p-4 flex flex-col items-center justify-center md:flex-row md:justify-between">
       <Link to="/"><H1 text="Book Swap Okapi Edition" /></Link>
+
+      <div className="flex items-center pt-2 gap-6 md:justify-start">
+        <div>
+          <NavLink to="/" className={({ isActive }) => isActive && "font-bold"}>
+          <H2 text="Available books" /></NavLink>
+        </div>
```

Comment:
I wonder if you could make an atom for this? Since you're repeating the same class function in each NavLink, it might be nice to have your own dedicated component that gives you this